lib/child_process.js

```diff
@@ -245,11 +245,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
   }
 
   if (child.stdout) {
-    if (encoding)
-      child.stdout.setEncoding(encoding);
-
     child.stdout.addListener('data', function(chunk) {
-      stdoutLen += chunk.length;
+      stdoutLen += chunk.byteLength;
```

Comment:
@mscdex Maybe not. All things being equal, though, I guess I prefer `byteLength` because it's explicit about what is being measured.